### PR TITLE
Add config.project to changelog rake task

### PR DESCRIPTION
**Pull request by alexjfisher**
This is needed since
https://github.com/skywinder/github-changelog-generator/pull/451 was
merged.

### CODE REVIEW

File: moduleroot/Rakefile.erb [hunk 1]
```diff
@@ -56,6 +56,9 @@ begin
     config.header = "# Changelog\n\nAll notable changes to this project will be documented in this file.\nEach new release typically also includes the latest modulesync defaults.\nThese should not affect the functionality of the module."
     config.exclude_labels = %w{duplicate question invalid wontfix wont-fix modulesync skip-changelog}
     config.user = 'voxpupuli'
+    metadata_json = File.expand_path('./metadata.json', File.dirname(__FILE__))
+    metadata = PSON.load(File.read(metadata_json))
```
Comment (ekohl) [inline moduleroot/Rakefile.erb]: Why PSON and not JSON?

File: moduleroot/Rakefile.erb [hunk 2]
```diff
@@ -56,6 +56,9 @@ begin
     config.header = "# Changelog\n\nAll notable changes to this project will be documented in this file.\nEach new release typically also includes the latest modulesync defaults.\nThese should not affect the functionality of the module."
     config.exclude_labels = %w{duplicate question invalid wontfix wont-fix modulesync skip-changelog}
     config.user = 'voxpupuli'
+    metadata_json = File.expand_path('./metadata.json', File.dirname(__FILE__))
```
Comment (ekohl) [inline moduleroot/Rakefile.erb]: Does this work too and is it easier to read?

```ruby
metadata_json = File.join(File.dirname(__FILE__), 'metadata.json')
```

Comment (alexjfisher) [reply moduleroot/Rakefile.erb]: yes and yes. :)
Updated.

File: moduleroot/Rakefile.erb (hunk 1 above)
Comment (alexjfisher) [reply moduleroot/Rakefile.erb]: No good reason.  I just lazily copied from the stdlib load_module_metadata function.  Updated.

**Pull request merged**